### DISCUSSION TITLE
Include full path instead of current directory name

### DISCUSSION THREAD

**Discussion by tasdendu** [starship/starship#2345] [Ideas]
Would look great if we can include the full path (PWD) instead of just showing the current working directory name.

**Comment by jflores-p** [comment]
Well, setting `fish_style_pwd_dir_length = 1` in the directory module will display the full path with abbreviations, change the "1" with the number you like to display that number of character when the prompt makes the contraction, [here](https://github.com/starship/starship/blob/b8e2c4309914b020d6264e8eb0032b58d098f2ec/src/modules/directory.rs#L401-L443) the test code with assertions.
Pd: I had to dig a bit into the code to find the variable, the documentation mentions it but doesn't show how to do it.

**Comment by cometsong** [comment]
A `custom` module can be made.

```
[custom.pwd]
command = "echo $PWD/"
when = "true"
style = "bold cyan"
#format = "[$output]($style) "
format = "[\\[$output\\]]($style) "
```
FYI My second `format` line also wraps the `output` in [ ].
